src/EventStore.Core/LogV3/EventTypeIdConverter.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace EventStore.Core.LogV3 {
+	/// Converts between EventTypeIds and their event number in the event types stream.
+	public static class EventTypeIdConverter {
```

Comment:
we are relying on LogV3SystemEventTypes.EventTypeInterval being 1 here
consider adding a test specifically for this class that will fail if the interval changes